.storybook/preview.js

```diff
@@ -1,3 +1,6 @@
 export const parameters = {
   actions: { argTypesRegex: '^on[A-Z].*' },
+  options: {
+    storySort: { order: ['Home', 'Visualizations'] },
```

Comment:
Is it missing `Building blocks` ?

Reply:
Stuff that aren't listed just get added to the end.

Reply:
Great :+1: